src/ui/create_game.html

```diff
@@ -28,7 +29,7 @@
         cache: true,
         success: function() {
           Loader.loadScripts([
-            'js/extern/jquery-ui.min.js',
```

Comment:
Comment on removing the JQuery-UI library from here:

I think that we're not actually using the JQuery-UI library in the create_game.html page. I checked through the commit history, and the inclusion of this library dates all the way back to commit 1ed3976011024b6c652ddabe78a116093bb5e4bb, back in April 2013. It was only added to create_game.html to enable autocomplete. In other places, it's used for the date picker.